src/apis/geolocation.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export default async (input) => {
+  try {
+    let coords = [];
+    const response = await axios.get(
+      `http://api.openweathermap.org/geo/1.0/direct?q=${input}&limit=1&appid=${
```

Comment:
It still needs to be HTTPS protocol. If we deploy this, it will throw an error when the website will be hosted on HTTPS and be calling to HTTP(not secure connection).

```suggestion
      `https://api.openweathermap.org/geo/1.0/direct?q=${input}&limit=1&appid=${
```